packages/polymath-ui/src/components/EthNetworkWrapper/index.js

```diff
@@ -3,21 +3,25 @@
 import EthNetworkWrapper from './EthNetworkWrapper';
 
 export { default as reducer } from './reducer';
-export {
-  CONNECTED,
-  ERROR_LOCKED,
-  ERROR_NETWORK,
-  ERROR_NOT_INSTALLED,
-  ERROR_DISCONNECTED,
-} from './actions';
+
+export const ERROR_NOT_INSTALLED = 1;
+export const ERROR_LOCKED = 2;
+export const ERROR_NETWORK = 3;
+export const ERROR_DISCONNECTED = 4;
+export const ERROR_ACCESS_REQUESTED = 5;
+export const ERROR_ACCESS_DENIED = 6;
+
+export const CONNECTED = 'polymath-auth/CONNECTED';
```

Comment:
I'm not not super convinced this is a good idea of defining/exporting code (I know it was being done this way) coming from actions in `EthNetworkWrapper/index.js`. Shouldn't these come from the `actions.js` file?

Reply:
At least I think these constants are more coupled to the action creators than a specific component (even if they were made explicitly for such component). This is also  what I've at least seen as common practice

Reply:
Yeah in this context I agree as the component doesn't make much sense anyway with these Redux actions that are not really encapsulated/tighten to the component...